src/GraphUtils.ts

```diff
@@ -401,42 +397,21 @@ export function findTurnPts(pts: Point[], mode: string) {
             }
         }
         if (!isDefined(position)) continue;
-        if (statPts[i][1] < pts[position-5][1] && statPts[i][1] < pts[position+5][1]) {
+        if (statPts[i][1] < pts[mod(position-5, pts.length)][1] && statPts[i][1] < pts[mod(position+5, pts.length)][1]) {
```

Comment:
```suggestion
        if (statPts[i][1] > pts[mod(position-5, pts.length)][1] && statPts[i][1] > pts[mod(position+5, pts.length)][1]) {
```

As this should be a maxima